server/container/guice/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/JmapGuiceProbe.java

```diff
@@ -66,8 +67,8 @@ public Port getJmapPort() {
         return jmapServer.getPort();
     }
 
-    public void addMailboxListener(MailboxListener.GroupMailboxListener listener) {
-        eventBus.register(listener);
+    public void addMailboxListener(MailboxListener listener, RegistrationKey registrationKey) {
```

Comment:
ok but why?

Reply:
Because it works.

And because I want to register all groups at startup to be able to detect changes.

Reply:
please add the explanation as a commit message  or a code comment